src/DayColumn.js

```diff
@@ -210,6 +210,7 @@ class DayColumn extends React.Component {
     return styledEvents.map(({ event, style }, idx) => {
       let end = accessors.end(event)
       let start = accessors.start(event)
+      let key = accessors.eventId(event) || 'evt_' + idx
```

Comment:
Think, to be on the safe side, you should use `??` here, rather than `||`

Reply:
fixed! thanks cutter